### ISSUE TITLE
video has wrong scaling on iphone6

### ISSUE THREAD

**Issue by MaxBoehme** [phoboslab/jsmpeg#56]
I have the jsmpeg-video playing in a canvas which has its css-width/-height set to 100%  (of the containing box)
jsmpeg then sets automatically the dimensions of the video (1280/720 in this case) into the
width-/height/attribute of the canvas

BUT :
on iphones the video is somehow scaled up! I only see a detail of the actual video.
a little fix was setting the width-/height/attribute of the canvas again to the values,
BUT
that does not work on iphone6!

Any suggestions are well appreciated!
thanks.

**Comment by andyearnshaw** [comment]
Are you using the iOS 9 Beta? Someone raised a similar issue to me recently but noted that it only affects Safari on iOS 9, which isn't out yet.

**Comment by MaxBoehme** [comment]
No, it's the 8.4-version.

In the meanwhile I tried to narrow the problem down and it seems to be like
a lazy-rending-problem:
The video is showing in the wrong zoom (the canvas has the right size
inside the page though), but as soon as you scroll a little bit it is
fixing the videos zoom. Like the mobile browser doesn't redraw/repaint the
view of the canvas.
I was able to use a method like:
element.offsetHeight;
to force a redraw on an iphone4, but that still does not work on the
iphone6. Maybe that brings up an idea in your head, I am already quite
hopeless. (Still have to fix it, though...)

Thanks for the reply! I'll let you know if I find a solution.
